src/ServiceControl.UnitTests/API/APIApprovals.cs

```diff
@@ -76,5 +76,16 @@ public void RootPathValue()
             Assert.AreEqual(HttpStatusCode.OK, result.StatusCode);
             Approver.Verify(JToken.Parse(result.Body.AsString()).ToString(Formatting.Indented));
         }
+
+        [Test]
+        public void TransportNames()
```

Comment:
This entire test needs to have the "transprot" typo corrected in several places.

Reply:
done